src/picongpu/include/particles/gasProfiles/GaussianCloudImpl.hpp

```diff
@@ -64,13 +63,15 @@ struct GaussianCloudImpl : public T_ParamClass
 
         if (globalCellPos.y() < vacuum_y) return float_X(0.0);
 
-        float_X density(1.0);
-        const float_X power = ParamClass::gasPower;
-        for (uint32_t d = 0; d < simDim; ++d)
-        {
-            const float_X exponent(math::abs((globalCellPos[d] - center[d]) / sigma[d]));
-            density *= math::exp(ParamClass::gasFactor * math::pow(exponent, power));
-        }
+        /* for x, y, z calculate: x-x0 / sigma_x */
+        const floatD_X r0overSigma = (globalCellPos - center) / sigma;
```

Comment:
Could you please multiply this with `sqrt(0.5)`

The Gaussian distribution has a factor `exp(x^2/2/sigma^2)` we are missing.
Otherwise we should rename `sigma` to avoid confusion.